### DISCUSSION TITLE
DNS: Adding a new CAA record

### DISCUSSION THREAD

**Discussion by tlrjs** [vercel/community#543] [Help] [answered]
I'm trying to setup an `api.` subdomain. However, the CAA record I added on Vercel still doesn't show up 24hrs later when I lookup dns records. 

I added CAA `0 issue "digicert.com"` but when I do a dns lookup it only shows the letsencrypt.org CAA record. I'm using Render.com for my backend server that the `api` subdomain will point to and Render.com requires a digicert.com CAA record.

**Comment by swarnava** [comment, marked as answer]
Hi,

We need to take a closer look at your account to check your domain configuration and other details. Could you raise a request here: https://vercel.com/support/request

**Reply by tlrjs** [reply]
Thanks, case (#00076481) submitted.
